microsoft-edge/webdriver-chromium.md

```diff
@@ -46,7 +84,17 @@ Now, [download the matching version of Microsoft Edge Driver][MicrosoftDeveloper
 
 > [!NOTE]
 > Microsoft Edge \(EdgeHTML\) does not work with [Microsoft Edge Driver][MicrosoftDeveloperEdgeToolsWebdriverDownloads].  To automate Microsoft Edge \(EdgeHTML\), you must download [Microsoft WebDriver for Microsoft Edge \(EdgeHTML\)][Webdriver].  
+-->  
+
+# Choose a WebDriver language binding  
 
+The last component you must download is a language-specific client driver to translate your code \(Python, Java, C\#, Ruby, JavaScript\) into commands the Microsoft Edge Driver runs in Microsoft Edge \(Chromium\).  
+[Download the WebDriver language binding of your choice][SeleniumDownloads].  The Microsoft Edge team recommends [Selenium 4.00-alpha05][NugetPackagesSeleniumWebdriver400alpha05] or later, because it supports Microsoft Edge \(Chromium\).  However, you are able to control Microsoft Edge \(Chromium\) in all older versions of Selenium, including the current stable Selenium 3 release.  
```

Comment:
The spacing between paragraphs here is rendering strangely to me in the preview: 
![image](https://user-images.githubusercontent.com/34316555/84063217-8ec89800-a975-11ea-9d3d-120205d96a2b.png)

Reply:
Updated.